docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: '3'
+
+services:
+  mongo_host:
+    image: "mongo"
+    container_name: mongo_host
+    # restart: always
+    volumes:  
+      - dbdata:/data/db   
+    # networks:
+    #   - app-network 
+    ports:
+      - "37017:27017"
```

Comment:
Why change the port? What does this do for you?

Reply:
I have one mongo image but im trying to spin two different containers because my services use mongo and im running them in the same ec2 instance. To do so, I need to use a different port for one of the mongo containers. Also, another mongo container name!